Makefile.am

```diff
@@ -21,7 +21,9 @@ include_HEADERS = \
  include/margo-logging.h \
  include/margo-diag.h \
  include/margo-bulk-pool.h \
- include/margo-util.h
+ include/margo-util.h \
+ include/margo-bulk-util.h \
+ include/margo-timer.h
```

Comment:
that header lives in `src/margo-timer.h`  maybe it needs to move to include?

Reply:
It does not, there is a margo-timer-private.h in src, margo-timer.h is in include...

Reply:
withdrawn! i forgot to rebase my 'main' onto latest. thanks.